qemu/tests/cfg/fio_perf.cfg

```diff
@@ -0,0 +1,78 @@
+- fio:
+    only x86_64
+    type = fio_perf
+    numa_node = -1
+    block_size = "4k 16k 64k 256k"
+    format = False
+    iodepth = "1 8 64"
+    threads = "16"
+    rw = "read write randread randwrite randrw"
+    variants:
+        - localfs:
+            image_name_disk1 = /mnt/storage2
+        - nfs_block:
+            image_name_disk1 = images/storage2
+        - fusion-io:
+            image_name_disk1 = /dev/fioa
+    variants:
+        - file_system:
+            image_size_disk1 = 40G
+            force_create_image_disk1 = yes
+            remove_image_disk1 = yes
+            format = True
+        - raw_disk:
+            image_raw_device_disk1 = yes
+            vhost_nic1 =
+    variants:
+        - single_disk:
+            images += " disk1"
+
+    variants:
+        - fio_linux:
+            only Linux
+            pattern = ".*[read|write].*bw=(\d+(?:\.\d+)?[\w|\s]B/s),\siops=(\d+).*"
+            order_list = "Block_size Iodepth Threads BW(MB/S) IOPS Latency(ms) Host_CPU BW/CPU KVM_Exits Util%"
+            guest_ver_cmd = "uname -r"
+            compile_cmd = "make && make install"
+            pre_cmd = "i=`/bin/ls /dev/[vs]db` && mkfs.xfs $i > /dev/null; partprobe; umount /mnt; mount $i /mnt"
+            drop_cache = "sync && echo 3 > /proc/sys/vm/drop_caches"
+            guest_result_file = /tmp/fio_result
+            variants:
+                - dedicated_mach1:
+                    mac_nic1 = 9a:37:37:37:37:4e
```

Comment:
Because we may hit the problem which could not get the ip from dhcp server on the VM. And we usually run our test in the evening or at weekends. The problem really influence our work, so after discussion with  eng ops, they assign some mac for us to get permanent ip.